pkg/universal-cnf/ucnf/ucnf_nse.go

```diff
@@ -36,7 +36,9 @@ func NewUcnfNse(configPath string, verify bool, backend config.UniversalCNFBacke
 		}
 	}()
 
-	err = nseconfig.NewConfig(yaml.NewDecoder(f), cnfConfig)
+	decodedYAML := yaml.NewDecoder(f)
+	decodedYAML.SetStrict(true) // enable strict decoding to help debug potential issues for decoding YAMLs
+	err = nseconfig.NewConfig(decodedYAML, cnfConfig)
 	if err != nil {
 		logrus.Warningf("NSE config errors: %v", err)
```

Comment:
Should we keep warning when decoding fails and not rather return error? @tiswanso is there a reason why we would want to continue execution when the config is not valid?

Reply:
it'd be best to make the `SetStrict` is controllable via an env var